src/browser/main.js

```diff
@@ -842,10 +842,36 @@
             {
                 $("boot_options").style.display = "none";
 
+                parse_param()
                 start_emulation(settings, done);
             }
         }
 
+        function parse_param()
```

Comment:
Could you pass settings to this function and move it to the outer scope (above `onload`)?